package.json

```diff
@@ -49,7 +49,7 @@
     "fix:license": "gulp update-license-headers",
     "clean": "rm -rf .go_workspace .tmp coverage dist npm-debug.log",
     "postversion": "node aio/scripts/version.js",
-    "postinstall": "node aio/scripts/version.js && ./node_modules/protractor/bin/webdriver-manager update --gecko=false"
+    "postinstall": "node aio/scripts/version.js && ./node_modules/protractor/bin/webdriver-manager update --gecko=false && go get -u github.com/golangci/golangci-lint/cmd/golangci-lint"
```

Comment:
`${GOPATH}/bin` may be not included in `${PATH}` in Travis CI.

Reply:
Because https://github.com/kubernetes/dashboard/blob/43c9568a08e844a70518f5c958e6ca689a96f950/.travis.yml#L19 changes `${GOPATH}`, I guess.